tests/vexflow_test_helpers.js

```diff
@@ -213,7 +209,7 @@ VF.Test = (function () {
     },
 
     runNodeTest: function (name, func, params) {
-      var fs = require('fs');
+      var fs = VF['fs'];
```

Comment:
Can we scope this to `VF['shims']['fs']` and also add a comment about it?

Reply:
done